docs/source/rules.rst

```diff
@@ -400,6 +400,13 @@ get details about a trigger instance by using ``get``.
 
   st2 triggerinstance get 556e135232ed35569ff23238
 
+Something that might be useful in debugging a rule is to re-send a trigger instance into |st2|. You
+can use the ``re_emit`` command for that.
+
+.. code-block:: bash
+
+  st2 triggerinstance re_emit 556e135232ed35569ff23238
```

Comment:
For consistency with `policy-type` and `action-alias`, resource name should be `trigger-instance` :panda_face: